### PR TITLE
Created Money Game

### PR DESCRIPTION
**Pull request by CalebSchni**

### CODE REVIEW

File: game.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class helping:
+  def menu():
+      print ("---------------------------------------------------------------")
+      print ('Lemonade Stand: cost $500 but you add $100 to your revenue, to buy answer ls.\n')
```
Comment (wesyao) [inline game.py:4]: Great work! I do think Lemonade Stands should give $125 revenue.. what do you think?

File: main.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+cr2 = []
+
+def buy(name,N):
+  answer = None
+  answers = ['ls','lm','rs','ss','hr','cr','ls2','lm2','rs2','ss2','hr2','cr2','lsa','lma','rsa','ssa','hra','cra','m']
+  while (answer is None) or (answer in answers):
+    answer = input('%s, What do you want to buy? Answer m to open the menu, or press enter if you do not want to buy anything.\n' % name)
+    if answer == 'ls' :
+      if ls2[N] == 1:
+        players[N].buying(200,500)
```
Comment (wesyao) [inline main.py:27]: Lets change this to $250 for doubling lemonade stands

File: main.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+      else:
+        players[N].buying(100,500)
```
Comment (wesyao) [inline main.py:29]: Lets change this to be $125 for single lemonade stands

**Review by wesyao — CHANGES_REQUESTED** [review]
Looks good, just make the small corrections and check it in

File: game.py (hunk 1 above)
Comment (CalebSchni) [reply game.py:4]: tat Is not the point I made a whole algorithm on how much stuff should cost starting with the lemonade stand

**Review by wesyao — APPROVED** [review]
i agree

**Pull request merged**